### PR TITLE
MC-553 CP Subsytem documentation updates

### PR DESCRIPTION
**Pull request by aberatcan**
- https://github.com/hazelcast/management-center/pull/5170

### CODE REVIEW

**Comment by Serdaro** [comment]
I am guessing https://docs.hazelcast.com/management-center/latest/getting-started/user-interface should also be included with the newly added item

**Comment by JakeSCahill** [comment]
I'll take a look through the content in the morning.

I see that the UI is inconsistent though. We should either use CP Subsystem or CPSubsystem, not both.

image

**Comment by utkukaratas** [comment]
thanks @JakeSCahill; i'll patch this. "CP Subsystem" seems the old and right choice.

**Comment by utkukaratas** [comment]
hi @JakeSCahill,

everything looks good except the navigation links under CPSubsystem > Dashboard are broken:

- Promoting Members to CP Members
- Removing CP Members
- Restarting CP Subsystem

**Comment by JakeSCahill** [comment]
Nice, yeah I'm working on some additions to the content along with some redirect rules.

**Comment by utkukaratas** [comment]
@JakeSCahill slack is down i'll answer here:

- version is the version number of the session, basically it shows how many times a session heartbeat is received.

- endpoint/endpointType is always 0 right now, as i write this i wonder why i haven't removed this. i will remove these two from that table.

**Comment by JakeSCahill** [comment]
Thanks 😄  

And 'Member' is the member that started the session? I read that clients can also start a session, so I was curious.

**Comment by utkukaratas** [comment]
> And 'Member' is the member that started the session? I read that clients can also start a session, so I was curious.

no, i poll each alive member for the metrics, as each member might report different session metric.

File: docs/modules/cp-subsystem/pages/dashboard.adoc [hunk 1]
```diff
@@ -108,6 +108,7 @@ members are missing.
 * *Majority of the CP members are not accessible:* The majority of CP
 members are missing.
 
+[#promote]
```
Comment (JakeSCahill) [inline docs/modules/cp-subsystem/pages/dashboard.adoc:111]:
```suggestion
[[promote]]
```

File: docs/modules/cp-subsystem/pages/dashboard.adoc [hunk 2]
```diff
@@ -126,6 +127,7 @@ image:ROOT:CPPromoteDialog.png[Promote Member to CP Confirmation]
 
 If an exception occurs, an error notification is displayed.
 
+[#remove]
```
Comment (JakeSCahill) [inline docs/modules/cp-subsystem/pages/dashboard.adoc:130]:
```suggestion
[[remove]]
```

File: docs/modules/cp-subsystem/pages/dashboard.adoc [hunk 3]
```diff
@@ -146,7 +148,7 @@ If everything goes well, a success message is displayed.
 image:ROOT:CPRemoveDialog.png[Remove CP Member Confirmation]
 
 If an exception occurs, an error notification is displayed.
-
+[#restart]
```
Comment (JakeSCahill) [inline docs/modules/cp-subsystem/pages/dashboard.adoc:151]:
```suggestion
[[restart]]
```

**Comment by JakeSCahill** [comment]
I'm adding some improvements to our Platform CP docs and will add some cross-links to surface this new functionality in MC.

**Comment by utkukaratas** [comment]
if https://github.com/hazelcast/hz-docs/pull/391 fixes the broken links in dashboard page, this seems good to go.

**Comment by JakeSCahill** [comment]
> if [hazelcast/hz-docs#391](https://github.com/hazelcast/hz-docs/pull/391) fixes the broken links in dashboard page, this seems good to go.

Broken links?

**Comment by aberatcan** [comment]
> > if [hazelcast/hz-docs#391](https://github.com/hazelcast/hz-docs/pull/391) fixes the broken links in dashboard page, this seems good to go.
> 
> Broken links?

The links that navigate to the hazelcast docs @JakeSCahill They are not working as expected

**Comment by utkukaratas** [comment]
@JakeSCahill e.g. 
image

**Comment by JakeSCahill** [comment]
> @JakeSCahill e.g. image

Ah this is just because we don't build the Platform docs for the MC staging site.

When this goes live, the links will work because the link checker GitHub Action passed 😄

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**